.vscode/cspell.json

```diff
@@ -323,6 +323,8 @@
         "esrp",
         "ESRPRELPACMANTEST",
         "eventgrid",
+        "eventgridtopicendpoint",
```

Comment:
Do we need these exceptions? They are used for environment variables from what I see... could put Event_Grid_Topic_Endpoint